modulefiles/wflow_gaea

```diff
@@ -12,4 +12,7 @@ module load rocoto
 module load miniconda3
 
 setenv CONDA_DEFAULT_ENV "regional_workflow"
-
+setenv PROJ_LIB /lustre/f2/pdata/esrl/gsd/contrib/miniconda3/4.8.3/envs/regional_workflow/share/proj
+if { [module-info mode load] } {
+   system "conda activate regional_workflow;"
```

Comment:
I have found that activating the regional_workflow environment in the same modulefile that miniconda3 is loaded doesn't work the first time you do `module load wflow_gaea` (conda init doesn't work). It works the second time you do it though. Thus, I chose to leave it  to the user to activate the environment manually with something like:
```
  puts stderr "Please do the following to activate conda:
       > conda activate regional_workflow"
```
The only system that direct activation of conda environment worked with a direct conda activate call is singularity, and I think that is because the miniconda3 modulefile there is written in Lua instead of TCL, which Lmod likes better. In most other systems it didn't work.